docs/source/install/details/database.rst

```diff
@@ -15,14 +15,14 @@ On Ubuntu and other Debian derivative distributions this can be accomplished wit
 
     $ sudo apt-get install postgresql postgresql-server-dev-all postgresql-client
 
-For Mac OS X, binary packages can be downloaded from the official website of `postgresql`_ or you can use ``brew``::
+For Mac OS X, binary packages can be downloaded from the official website of `PostgreSQL`_ or you can use ``brew``::
 
     $ brew install postgresql
     $ pg_ctl -D /usr/local/var/postgres start
 
 To manually create a database for AiiDA that will later be used in the configuration with ``verdi setup``, you should follow these instructions.
 First you will need to run the program ``psql`` to interact with postgres and you have to do so as the ``postgres`` user that was created upon installing the software.
-To assume the role of ``postgres`` run as root::
+To assume the role of ``postgres`` by run as root::
```

Comment:
Either "To assume ... **run as** root" or "Assume the role ... **by running as** root"

Reply:
yes, sorry, didn't actually mean to change that line